mgmtsystem_evaluation/security/ir.model.access.csv

```diff
@@ -5,3 +5,4 @@
 "manage_mgmtsystem_evaluation_template","mgmtsystem.evaluation.template","model_mgmtsystem_evaluation_template","mgmtsystem.group_mgmtsystem_user",1,1,1,0
 "access_mgmtsystem_evaluation_result","mgmtsystem.evaluation.result","model_mgmtsystem_evaluation_result","base.group_user",1,0,0,0
 "manage_mgmtsystem_evaluation_result","mgmtsystem.evaluation.result","model_mgmtsystem_evaluation_result","mgmtsystem.group_mgmtsystem_user",1,1,1,0
+"manage_mgmtsystem_evaluation_ir_model","ir.model","model_ir_model","mgmtsystem.group_mgmtsystem_user",1,0,0,0
```

Comment:
I wasn’t able to fix the issue without giving read access to `ir.model` for the `group_mgmtsystem_user`.  
The problematic fields appear to be `model_id` and `resource`.  
Do you happen to know a cleaner solution that avoids granting this permission? 
@etobella

Reply:
This is not an option